src/themes/masschallenge/collections/grid.overrides

```diff
@@ -65,4 +65,9 @@
 
 .ui.zero-margin.grid {
   margin: 0;
+}
+
+.ui.grid > .row.link-row:hover {
+  background: @lightGray;
+  cursor: pointer
```

Comment:
Instead of extending `.ui.grid`, Semantic already has a list that has a hover state. I'm pretty sure this will work out of the box. https://react.semantic-ui.com/elements/list/#variations-selection